### PR TITLE
active page bug fixed 

### PR DESCRIPTION
**Pull request by tusharrao198**
Issue #151 
1. now the current tab is active on which page you are on.
2. added .active tag in style.css 
3. style.css moved to css folder

### CODE REVIEW

File: src/stac_iitmandi/static/stac_iitmandi/css/style.css [hunk 1]
```diff
@@ -11,7 +11,15 @@ li {
 }
 
 .bg-custom {
-    background-color: white;
+    background-color: #FFFFFF;
+    border-radius: 10px;
+}
```
Comment (LalitNM) [inline src/stac_iitmandi/static/stac_iitmandi/css/style.css:16]: This class is useless, you can anyway use `bg-white` or `bg-light` to achieve the same.

File: src/stac_iitmandi/templates/stac_iitmandi/_navbar.html [hunk 2]
```diff
@@ -6,31 +6,31 @@
...
-                    <li><a class="dropdown-item" href={% url 'zenith' %}>ZENITH</a></li>
-                    <li><a class="dropdown-item" href={% url 'utkarsh' %}>UTKARSH</a></li>
+                    <li><a class="dropdown-item {{astrax}}" href={% url 'astrax' %}>ASTRAX</a></li>
+                    <li><a class="dropdown-item {{zenith}}" href={% url 'zenith' %}>ZENITH</a></li>
+                    <li><a class="dropdown-item {{utkarsh}}" href={% url 'utkarsh' %}>UTKARSH</a></li>
                 </ul>
             </li>
             <li class="nav-item">
-                <a class="nav-link" href="#blogsite">Blog</a>
+                <a class="nav-link" href="#http://blog.pc.iitmandi.co.in/">Blog</a>
```
Comment (LalitNM) [inline src/stac_iitmandi/templates/stac_iitmandi/_navbar.html:21]: Ideally this should be linked to [blog.stac.iitmandi.co.in](blog.stac.iitmandi.co.in). But for now you can leave it, we will add all links a the end of project.

**Review by LalitNM — CHANGES_REQUESTED** [review]

**Comment by LalitNM** [comment]
In addition to above changes, you also need to change the active class for drop-down items. This should be styled in a way similar to 'Home'. A screenshot is attached
active
This blue color should be changed.

**Comment by divyanshvinayak** [comment]
> In addition to above changes, you also need to change the active class for drop-down items. This should be styled in a way similar to 'Home'. A screenshot is attached
> active
> This blue color should be changed.

Guys, if it looks like this, then I guess it is awesome enough

File: src/stac_iitmandi/static/stac_iitmandi/css/style.css (hunk 1 above)
Comment (tusharrao198) [reply src/stac_iitmandi/static/stac_iitmandi/css/style.css:16]: basically, i added that for boder-radius

Comment (LalitNM) [reply src/stac_iitmandi/static/stac_iitmandi/css/style.css:16]: Have you tried `rounded-sm` class of bootstrap?

**Comment by tusharrao198** [comment]
> In addition to above changes, you also need to change the active class for drop-down items. This should be styled in a way similar to 'Home'. A screenshot is attached
> active
> This blue color should be changed.

Do I need to change this color ? It looks fine.

File: src/stac_iitmandi/static/stac_iitmandi/css/style.css (hunk 1 above)
Comment (tusharrao198) [reply src/stac_iitmandi/static/stac_iitmandi/css/style.css:16]: Will try.

**Comment by tusharrao198** [comment]
Do i still need to change active color?
It looks fine by the way

**Comment by LalitNM** [comment]
@tusharrao198 as suggested by @divyanshvinayak that this color looks awesome, leave it for now. Do other requested changes.

**Review by LalitNM — APPROVED** [review]

**Comment by LalitNM** [comment]
This looks great 👏 merging this.

**Pull request merged**